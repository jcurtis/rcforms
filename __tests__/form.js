import React from 'react'
import renderer from 'react-test-renderer'
import Form from '../lib/index.js'

test('Form renders', () => {
  const component = renderer.create(
    <Form />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
