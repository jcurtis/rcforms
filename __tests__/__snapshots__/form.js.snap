// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders 1`] = `
<form
  className="form"
  noValidate={true}
  onSubmit={undefined}
  role="form"
/>
`;
